src/fu-udev-backend.c

```diff
@@ -487,6 +490,33 @@ fu_udev_backend_netlink_parse_blob(FuUdevBackend *self, GBytes *blob, GError **e
 		/* success */
 		fu_udev_backend_device_add_from_device(self, device_actual);
 	}
+#else
```

Comment:
do we already have a linux CI target running without systemd?  Maybe OpenBMC should explicitly

Reply:
Does the macos build count?